src/App.tsx

```diff
@@ -9,39 +9,29 @@
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
+// import { getUsers } from './api/users';
+// import { set } from './features/users';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { closePost, init, setSelectedPost } from './features/posts';
 
 export const App: React.FC = () => {
-  const [posts, setPosts] = useState<Post[]>([]);
-  const [loaded, setLoaded] = useState(false);
-  const [hasError, setError] = useState(false);
+  // const [loaded, setLoaded] = useState(false);
+  // const [hasError, setError] = useState(false);
```

Comment:
These lines are commented out. If the `useState` hooks are not needed, consider removing them. If they are needed, ensure they are correctly implemented and used in the code.